clusters/hive/pools/serverless/install-config-aws-us-east-1_secret.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: install-config-aws-us-east-1
+  namespace: serverless-cluster-pool
+stringData:
+  install-config.yaml: |
+    apiVersion: v1
+    baseDomain: serverless.devcluster.openshift.com
+    compute:
+    - architecture: amd64
+      hyperthreading: Enabled
+      name: worker
+      platform:
+        aws:
+          type: m5.xlarge
```

Comment:
Does are [spot instances](https://aws.amazon.com/ec2/spot)?

Reply:
AFAIK, these are NOT spot instances. I think using spot instances in this PR now is probably not a good idea (if it's possible at all). We should first test it somewhere else and see how reliable it is. But it's an interesting idea. Worth trying.

Reply:
Using spot instances for testing would bring massive savings. Also, they are actually targeting CI/CD.

UI agrees we should start with Spot instances, but we definitely should migrate to it. That would probably require fixing some tests, so they may tolerate K8s' node failure. That most likely needs to be fixed upstream. In fact, I think we could use [GKE spot instances](https://cloud.google.com/kubernetes-engine/docs/concepts/spot-vms) on upstream, too. I just opened an issue for this: https://github.com/knative/infra/issues/384

When using spot instances, we should have more, but smaller nodes.